File: C++/Leetcode 1470.cpp
#include <stdio.h>
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int> vect;

        int j = n;
        for (int i = 0; i < n; i++)
        {
            vect.push_back(nums[i]);
            vect.push_back(nums[j]);
            j++;
        }
        return vect;
    }
};

File: Python/Leetcode 1470.py
from ast import List

class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        Final_List = []

        j = n

        for i in range(n):
            Final_List.append(nums[i])
            Final_List.append(nums[j])
            j = j + 1
        
        return Final_List